templates/tarsnap-archive.erb

```diff
@@ -3,4 +3,4 @@
 title="$1"
 shift
 
-<%= @path %> --quiet -c -f $title.$(date +%Y%m%d%H%M) $@
+<%= scope.lookupvar("tarsnap::path") %> --quiet -c -f $title.$(date +%Y%m%d%H%M) $@
```

Comment:
@igalic Tried to explicitly `include ::tarsnap`, but it doesn't work for me. (Running Puppet master 3.7.4 and Puppet agent 3.6.2.)